include/clspv/Passes.h

```diff
@@ -25,6 +25,47 @@ class ModulePass;
 class raw_pwrite_stream;
 class raw_ostream;
 template <typename T> class ArrayRef;
+
+// Clspv pass initializers.
+class PassRegistry;
+
+// Initialize all clspv passes.
+void initializeClspvPasses(PassRegistry &);
+
+// Individual pass initializers.
+void initializeAllocateDescriptorsPassPass(PassRegistry &);
```

Comment:
Do these individual pass initializers need to be exposed in the public header?

Reply:
In some header available to `clspv-opt`.

Reply:
Oh, and available to the passes too.  The macro expands to code that defines the `initialize*` function, so the declaration needs to be available.  `clspv/Passes.h` seemed the best home for these.

Reply:
Doesn't just the overall initializer need to be available to clspv-opt? Could we have a header local to lib/ that declares the individual pass initializers?

Reply:
That should work.  Yes.